Jenkinsfile

```
@@ -357,7 +357,16 @@ pipeline {
                 }
             }
         }
+        stage('Benchmarks'){
+            steps{
+                withEnv(["PATH+=${GO}:${GOPATH}/bin"]){
+                    warnError(message: "one or more benchmarks failed") {
+                        sh "go test -timeout=20m -count=1 -run=- -bench=. -benchmem -v ${SGW_REPO}/... | tee benchmark.out || true"
```

We want this to return a non-zero exit code in the case of a failure for the `warnError` wrapper to pick up.

```suggestion
                        sh "go test -timeout=20m -count=1 -run=- -bench=. -benchmem -v ${SGW_REPO}/... | tee benchmark.out"
```